dice.js

```diff
@@ -0,0 +1,21 @@
+function dice(dice =2){
```

Comment:
Great idea; you allow named parameters to be initialized with default values if no value or undefined is passed.